sdcm/microbenchmarking.py

```diff
@@ -150,12 +150,18 @@ def get_diffs():
                 "Diff last [%]": diff_last,  # diff in percents
                 "Diff best [%]": diff_best,
                 "has_regression": False,
+                "has_enhancement": False,
```

Comment:
better to say `improvement` and not enhancement

Reply:
Fixed